node/src/cli.rs

```diff
@@ -95,6 +96,16 @@ pub struct Cli {
     /// Relay chain arguments
     #[arg(raw = true)]
     pub relaychain_args: Vec<String>,
+
+    /// Maximum duration in milliseconds to produce a block
+    #[clap(long, default_value = "1500", value_parser=block_authoring_duration_parser)]
+    pub block_authoring_duration: Duration,
+}
+
+fn block_authoring_duration_parser(s: &str) -> Result<Duration, String> {
+    Ok(Duration::from_millis(clap_num::number_range(
+        s, 250, 2_000,
```

Comment:
what does both number mean?

Reply:
min and max allowed value. (I took it from moonbeam, not sure if we should credit it anywhere?)